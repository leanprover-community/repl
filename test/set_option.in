{"cmd": "#check 1"}

{"cmd": "set_option pp.numericTypes true\n#check 1"}

{"cmd": "#check 1", "setOptions":[[["pp","numericTypes"],true]]}

