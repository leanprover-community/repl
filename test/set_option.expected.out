{"messages":
 [{"severity": "info",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 6},
   "data": "1 : Nat"}],
 "env": 0}

{"messages":
 [{"severity": "info",
   "pos": {"line": 2, "column": 0},
   "endPos": {"line": 2, "column": 6},
   "data": "(1 : Nat) : Nat"}],
 "env": 1}

{"messages":
 [{"severity": "info",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 6},
   "data": "(1 : Nat) : Nat"}],
 "env": 2}

